src/table/Table.ts

```diff
@@ -166,6 +168,7 @@ export function asTableFromArray(data: any[][], options: IAsTableOptions = {}):
   const columns = cols.map((col, i) => {
     return {
       name: col,
```

Comment:
Same in lin 192+193 (https://github.com/phovea/phovea_core/pull/83/files/b516b4dbbf10a879ff74c2fde45b9223bb10183c#diff-936ef1d331fb0454c0e0fececaa4e8edR192)